docs/src/main/asciidoc/dynamodb-guide.adoc

```diff
@@ -0,0 +1,751 @@
+                .key(key)
+                .attributesToGet(FRUIT_NAME_COL, FRUIT_DESC_COL)
+                .build();
+    }
+}
+----
+
+Then, create a `org.acme.dynamodb.FruitSyncService` that will be the business layer of our application and store/load the fruits from the DynamoDB using synchronous client.
+
+[source,java]
+----
+package org.acme.dynamodb;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+@ApplicationScoped
+public class FruitSyncService extends AbstractService {
+
+    @Inject
+    DynamoDbClient dynamoDB;
+
+    public List<Fruit> findAll() {
+        return dynamoDB.scanPaginator(scanRequest()).items().stream()
+                .map(Fruit::from)
+                .collect(Collectors.toList());
+    }
+
+    public List<Fruit> add(Fruit fruit) {
+        dynamoDB.putItem(putRequest(fruit));
+        return findAll();
+    }
+
+    public Fruit get(String name) {
+        return Fruit.from(dynamoDB.getItem(getRequest(name)).item());
+    }
+}
+----
+
+Now, edit `org.acme.dynamodb.FruitResource` class as follows:
+
+[source,java]
+----
+package org.acme.dynamodb;
+
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/fruits")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class FruitResource {
+
+    @Inject
+    FruitSyncService service;
+
+    @GET
+    public List<Fruit> getAll() {
+        return service.findAll();
+    }
+
+    @GET
+    @Path("{name}")
+    public Fruit getSingle(@PathParam("name") String name) {
+        return service.get(name);
+    }
+
+    @POST
+    public List<Fruit> add(Fruit fruit) {
+        service.add(fruit);
+        return getAll();
+    }
+}
+----
+
+The implementation is pretty straightforward and you just need to define your endpoints using the JAX-RS annotations and use the `FruitSyncService` or `FruitAsyncService` to list/add new fruits.
+
+== Configuring DynamoDB clients
+
+Both DynamoDB clients (syn & async) are configurable via the `application.properties` file that can be provided in the `src/main/resources` directory.
+
+If you're going to use local DynamoDB instance configure set it with:
+
+[source,properties]
+----
+quarkus.dynamodb.region=eu-central-1
+quarkus.dynamodb.endpoint-override=http://localhost:8000
+
+quarkus.dynamodb.credentials.type=STATIC
+quarkus.dynamodb.credentials.static-provider.access-key-id=test-key
+quarkus.dynamodb.credentials.static-provider.secret-access-key=test-secret
+----
+
+- `quarkus.dynamodb.region` - It's required by the client, but since you're using local DynamoDB instance you can pick any valid AWS region.
```

Comment:
```suggestion
- `quarkus.dynamodb.region` - It's required by the client, but since you're using a local DynamoDB instance you can pick any valid AWS region.
```